library/src/main/java/com/otaliastudios/zoom/AbsolutePoint.kt

```diff
@@ -0,0 +1,49 @@
+package com.otaliastudios.zoom
+
+/**
+ * This class represents an absolute point on the ZoomEngine canvas (or beyond it's bounds)
+ */
+data class AbsolutePoint(
```

Comment:
These make everything cleaner, thanks.
Are they exposed? If not let's make them internal.

Reply:
As `panX` and `panY` are exposed I also exposed `pan` which is an `AbsolutePoint`. I can. remove that if it should only be an internal representation but I think the easier handling of points can also be beneficial to devs using this library - don't you think?

Reply:
Yes, cool!